### ISSUE TITLE
Add ArtefactObs and ArtefactFrq to SNV view

### ISSUE THREAD

**Issue by mathiasbio** [Clinical-Genomics/scout#4937]
**Is your feature request related to a problem in the current program to new available techology or software? Please describe and add links/citations if appropriate.**

In the upcoming release of balsamic we are planning to add this feature: https://github.com/Clinical-Genomics/BALSAMIC/pull/1481

Which will filter variants which exist above 4 of 29 merged WGS normal samples. The tags in the info field of the VCF look as follows:

<img width="270" alt="image" src="https://github.com/user-attachments/assets/24ee4be0-a2f3-4d21-aa3d-073b507af427">

**Describe the solution you'd like**

It would be nice if this information also shows up in Scout in the this variant view:

<img width="426" alt="image" src="https://github.com/user-attachments/assets/96882b39-ea8a-4b88-88cf-3d3637252a40">

And here: 

<img width="280" alt="image" src="https://github.com/user-attachments/assets/157eda3a-2349-42e3-ae9a-c43b3fc62420">

And if possible, to add a filter option as well along with the currently implemented loqusdb annotations: 

<img width="466" alt="image" src="https://github.com/user-attachments/assets/b08c3e9b-50ff-4ddf-b631-f9c47eb1706f">

**Describe alternatives you've considered**

**Additional context**

**Comment by dnil** [comment]
Looks like you have a test case up there! Could you point to it here or on slack so we have a something to test with?

**Comment by mathiasbio** [comment]
Yes! I have 3 cases uploaded here which should have some variants annotated with this info: https://scout-stage.scilifelab.se/cust098/  such as grownshrew at position 1: 11187618

**Comment by northwestwitch** [comment]
Is this still actual @mathiasbio?

**Comment by mathiasbio** [comment]
Yes it seems so! I don't think it's very important, especially now as we have the other loqusdbs with panel observations, but it could be nice to highlight if it has been seen as an artefact in another dataset as well. It's only that we haven't at all set up the loqusdb connections here, and I think it actually only exists in stage, I then only used the exported database to annotate and filter in balsamic as a static file. So it's all "archived"
